### PR TITLE
Add Github Actions to run tests

### PR DESCRIPTION
**Pull request by rkdarst**
- Very minimal, using the suggested action template for a Python
  action
- Must set PYTHONPATH when running pytest since yaml2ics is in
  current directory.
- Review: automerge if tests pass :)

### CODE REVIEW

**Comment by rkdarst** [comment]
I guess it doesn't run tests until it's already there...

**Comment by rkdarst** [comment]
Well, they won't pass until #9.

File: .github/workflows/python-app.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```
Comment (stefanv) [inline .github/workflows/python-app.yml:4]:
```suggestion
name: Test yaml2ics
```

**Comment by stefanv** [comment]
Thanks, Richard!

**Comment by rkdarst** [comment]
great!

**Pull request merged**